### PR TITLE
Update citations in README.rst

### PR DESCRIPTION
**Pull request by maxnoe**
The list kept getting out of date and by design, it is impossible to have the correct doi in the README for the release itself.

So I replaced the list of Zenodo records with a link which lists all versions on zenodo and the latest doi again.

I also added the ICRC proceeding.

### CODE REVIEW

**Review by kosack — APPROVED** [review]

File: README.rst [hunk 1]
```diff
@@ -43,16 +27,30 @@ expert or developer!
...
+      doi = {10.22323/1.395.0744},
+      booktitle = {Proceedings, 37th International Cosmic Ray Conference},
+      year=2021,
+      volume={395},
+      number={744},
+      location={Berlin, Germany},
+    }
+
 
 Installation for Users
```
Comment (HealthyPear) [inline README.rst:55]: I think we don't really need this part in the README when we can just point to the actual documentation

**Review by HealthyPear — APPROVED** [review]

File: README.rst (hunk 1 above)
Comment (HealthyPear) [reply README.rst:55]: (I mean the installation instructions - from the preview it seems I refer to the citation)

Comment (kosack) [reply README.rst:55]: The documentation is more for developers, not for users (though probably that should be updated at some point to have both)

Comment (maxnoe) [reply README.rst:55]: Let's merge this and I'll make another PR updating the installation instructions

**Review by nbiederbeck — APPROVED** [review]

**Pull request merged**